CHANGES.rst

```diff
@@ -1,16 +1,20 @@
-Changes
-=======
+=========
+ Changes
+=========
 
 4.1.0 (unreleased)
-------------------
+==================
 
-- Add support for Python 3.5.
+- Add support for Python 3.5 and 3.6.
 
-- Drop support for Python 2.6.
+- Drop support for Python 2.6 and 3.3.
 
+- Host documentation at https://zopeprincipalregistry.readthedocs.io
+
+- Reach 100% test coverage and ensure we remain there.
 
 4.0.0 (2014-12-24)
-------------------
+==================
 
 - Add support for PyPy.  (PyPy3 is pending release of a fix for:
```

Comment:
As PyPy3 is now tested via TravisCI it could be mentioned in version 4.1 that it is supported now, too.

Reply:
OK, will do.